docs/tools/new_tools_temp.md

```diff
@@ -0,0 +1,144 @@
+
+Starting with version 6.x, Mbed OS is moving from Mbed CLI to a new build tool: <!--that has no proper name yet-->. It uses Ninja as a build system, and CMake to generate the build environment and manage the build process in a compiler-independent manner. <!--need better phrasing-->
+
+Mbed Tools parses the Mbed OS build configuration and outputs it to a format CMake can read. It also provides a use friendly interface to CMake and Ninja so you can configure, generte and execute builds with a single command.
+
+
+<span class="notes">**Note:** You'll still need Mbed CLI for older versions of Mbed OS (6.x and older). You can install both tools side by side.</span>
+
+# Install or upgrade
+
+Mbed Tools is a Python package called `mbed-tools`, so you can install it with pip.
+<!--other than being a super vague name, it's a problem that it's a single tool with a plural name. "Mbed Tools is"  just parses as an error!-->
+
+<span class="tips">**Tip:** We recommend using a virtual environment to avoid any Python dependency conflicts. We recommend the [Pipenv](https://github.com/pypa/pipenv/blob/master/README.md) tool to manage your virtual development environment.
+
+## Prerequisite
+
+- Python 3. Install for [Windows](https://docs.python.org/3/using/windows.html), [Linux](https://docs.python.org/3/using/unix.html) or [macOS](https://docs.python.org/3/using/mac.html).
+- Pip (if not included in your Python installation). [Install for all operating systems](https://pip.pypa.io/en/stable/installing/).
+- CMake. [Install version 3.18.1 or newer for all operating systems](https://cmake.org/install/).
+- Ninja [Install version 1.0 or newer for all operating systems](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages).
+<!--are there versions for cmake and ninja?-->
+
+## Install
+
+Use pip to install:
+
+- To install the latest release:
+
+    ```
+    python -m pip install mbed-tools
+    ```
+
+- To install a specific version:
+
+    ```
+    python -m pip install mbed-tools==<version number in major.minor.patch format>
+    ```
+
+- To install a pre-release or development version:
+
+    ```
+    python -m pip install mbed-tools --pre
+    ```
+
+## Upgrade
+
+Use pip to upgrade:
+
+```
+python -m pip install mbed-tools --upgrade
+```
+
+# Use
+
+To get help for a specific command, use `mbed-tools <command> --help`. For example, for helping with listing connected devices (the `devices` command):
+
+```
+mbed-tools devices --help
+```
+
+## Create a new project
+
+To create a new Mbed OS project in a specified path:
+
+- To create the project and download a new copy of Mbed OS (latest official release):
+
+    ```
+    mbed-tools init <PATH>
+    ```
+
+- To create a project without downloading a copy of Mbed OS (reuse an existing copy):
+
+    ```
+    mbed-tools init -c <PATH>
+    ```
+
+    <!--so how do I tell it where the existing copy is? In Mbed Studio you have to explicitly "tell it" where the local copy is-->
```

Comment:
The tool doesn't support "telling it where an existing copy is" yet, it would have to be symlinked into the program folder. We are going to support this at some point when we implement "workspace" support, but it's not done yet.

Reply:
How actually do I get an example going? How do I tell it to go download a copy of mbed-os-example-blinky, and do I need to use `init` in an example?

Reply:
How do we use SymLink, then? If it's not been done before in Mbed OS, we need to explain it.

Reply:
`init` is just for creating a new project, it's similar to `mbed new`. We also have a `clone` command for downloading examples, which is similar to `mbed import` but we haven't documented that command, yet. I think we should just say "copy" rather than symbolic linking as I'm not sure we've tested symlinked CMake builds work yet. The `-c` workflow with the `init` command won't be supported until we have workspace support in the tools, which isn't implemented yet.